index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html>
 <html>
-
 <head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>JS Sandbox</title>
-    <script src="js/index.js"></script>
     <link rel="stylesheet" type="text/css" href="css/index.css"/>
 </head>
 
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+</head>
```

Comment:
Something's not right here... the CSS stylesheet is included twice, and the `</head>` is there twice as well.